.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.16
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+      - run:
+          name: "Set up environment"
+          command: echo 'export PATH="$(npm bin):$PATH"' >> $BASH_ENV
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v2-dependencies-{{ checksum "plasma_framework/package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v2-dependencies-
+
+      - run:
+          name: "Install dependencies"
+          command: |
+            npm install
+            rm -rf node_modules/*/.git # workaround for problem with dependencies git
```

Comment:
can you explain a bit what is the issue here? or if there is links would be nice for me to understand this.

Reply:
If you try to update the node_modules when one of the deps is a git repo nmp thorws an error: shama/napa/issues/53.
Removing `.git` directories is a fast workaround.